tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "typeRoots": [
+            "node_modules/@types"
+        ],
+        "module": "commonjs",
+        "moduleResolution": "node",
+        "target": "es6",
+        "sourceMap": true,
+        "outDir": "build/",
+        "allowJs": true,
+        "rootDir": "src",
+        "allowUnreachableCode": true
```

Comment:
I removed "allowUnreachableCode" as it is not needed. The other options are necessary.